core/shared/src/main/scala/scalaz/zio/system/package.scala

```diff
@@ -1,12 +1,7 @@
 package scalaz.zio
 
-import java.time.Instant
-
 package object system {
 
-  /** Determine the current system time **/
-  val currentTime: IO[Nothing, Instant] = IO.sync(Instant.now)
```

Comment:
`java.time` isn't available in core Scala.js. It requires an additional dependency such as https://github.com/scala-js/scala-js-java-time (which is still "work in progress").

Reply:
I need this :) Please place it in a JVM-only file then...

Reply:
AFAIK `scala-js/scala-js-java-time` is more of a WIP in terms of stuff like locales and timezones etc. `Instance.now` is completely fine. Also, remember that this is just so that `java.time` links, it's already on the classpath due to Java 8.  I suggest adding this dependency.

Reply:
OK, adding in the dependency. Would be good to have this `shared` again.